.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
> Firstly, it is not actually needed.

It is if you want to use some of Babel's ES7 features, including the (commonly used) object rest and spread operators. I'd rather have access to features I use than have some strict rules I don't even use work properly.

> In the project there is 2 spaces indentation everywhere, so it is a little bit odd to see 4 spaces here (need .editorconfig #4)

This is a good point, but any time you make a change like indentation, it should be done in its own pull request so it's easy to see changes that have real meaning (such as rule changes, for instance).